test/extensions/transport_sockets/tls/integration/ssl_integration_test.cc

```diff
@@ -866,7 +870,7 @@ TEST_P(SslTapIntegrationTest, RequestWithTextProto) {
   TestUtility::loadFromFile(fmt::format("{}_{}.pb_text", path_prefix_, id), trace, *api_);
   // Test some obvious properties.
   EXPECT_TRUE(absl::StartsWith(trace.socket_buffered_trace().events(0).read().data().as_bytes(),
-                               "POST /test/long/url HTTP/1.1"));
+                               "GET /test/long/url HTTP/1.1"));
```

Comment:
OOC why does this change from POST to GET?

Reply:
we switched to using default request headers, which is a GET

Reply:
Ah, makes sense. Thanks